xml/cap_depl_eks.xml

```diff
@@ -36,11 +31,23 @@
 
   <para>
    When you create your cluster, use node sizes that are at least a
-   <literal>t2.large</literal>.
+   <literal>DS4_v2</literal>. The <literal>NodeVolumeSize</literal> must be a
+   minimum of 60GB.
+  </para>
+  <para>
+      Take note of special configurations that are required to successfully
+      deploy &productname; on AKS in <xref linkend="sec.cap.install-scf-eks"/>.
   </para>
-
   <para>
-   See <xref linkend="sec.cap.eks-iam"/> for guidance on configuring Identity
+      The instructions in this chapter provide general DNS requirements, and 
+      instructions for using the EKS load balancer to provide fault-tolerant 
+      access to your cluster. Your initial DNS configuration will map your
+      cluster domain to a single worker node, and then after the load balancer
```

Comment:
We should probably remove this sentence (initial DNS configuration mapping the cluster domain to a single worker node) as it is probably not accurate. Once the LB has been set up by EKS the kube public services get their own hostnames which then require CNAME mappings (*.domain, uaa.domain etc.) See @mook-as comments later.

Reply:
I changed to say first create the CNAMEs, then deploy, then after the hostnames are available edit the CNAMEs with the new hostnames. Is this correct?

Reply:
It may be a bit confusing to say create CNAMEs first...what we specify in the values YAML are the domain names (yourdomain.com) and FQDNs (uaa.yourdomain.com) - they don't resolve to anything until we create CNAME entries for them mapping them to actual hostnames. And those hostnames come into the picture only after ELB has created them. So at some point during UAA deployment ELB would have set up a hostname for the `uaa-public` service which you can grab by a `kubectl get svc -n uaa` and then create the CNAME entries for uaa.yourdomain.com and *.uaa.yourdomain.com mapping them to that hostname. After that we deploy SCF and SCF pods then can communicate with UAA using uaa.yourdomain.com which now resolves. Same thing for the SCF public service entries needs to be done after SCF services come up on the ELB.